src/components/notifications/GtxMessaging.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="mb-3">
+        <label for="gtxmessaging-api-key" class="form-label">{{ $t("gtxMessagingApiKey") }}</label>
+        <HiddenInput id="gtxmessaging-api-key" v-model="$parent.notification.gtxMessagingApiKey" :required="true"></HiddenInput>
```

Comment:
How easy is it to find this api key in their UI? does this need a helptext how this can be found?

Reply:
Added a hint